### ISSUE TITLE
layers-slices reports an import inside the same slice as a cross-layer import when the segment is not in the default list

### ISSUE THREAD

**Issue by conarti** [conarti/eslint-plugin-feature-sliced#41]
## Problem

When the folder after the slice is not one of `ui`, `model`, `api`, `lib`, `config`, slice extraction falls back to the last part of the path. The segment is then taken for a slice, and an import inside the same slice looks like a layer importing itself.

## Reproduction

`src/widgets/header/hooks/index.ts` exists, and `src/widgets/header/Header.tsx` contains:

```ts
import { other } from '@/widgets/header/hooks';
```

```
2:23  error  You cannot import layer "widgets" into "widgets" (shared -> entities -> features -> widgets -> pages -> processes -> app)  @conarti/feature-sliced/layers-slices  [can-not-import]
```

The same happens with a subpath alias such as `#widgets/header/hooks`.

## Expected

An import that stays inside the same slice is not a layer violation. The message is also misleading: it says a layer cannot be imported into itself, which is not what happened.

## Impact

Default preset. Measured on public projects: 38 of 182 `layers-slices` reports in one, 5 of 82 in another. Listing the folder in the `segments` setting silences it, but the default behaviour should not report an import inside a slice.

## Plugin version

`@conarti/eslint-plugin-feature-sliced@2.0.0`
